src/System.Net.Requests/tests/RequestStreamTest.cs

```diff
@@ -11,11 +11,18 @@
 
 namespace System.Net.Tests
 {
-    [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Tests hang on .NET Framework")]
     public class RequestStreamTest
     {
         readonly byte[] buffer = new byte[1];
 
+        static RequestStreamTest()
+        {
+            // On .NET Framework, the default limit for connections/server is very low (2).
+            // On .NET Core, the default limit is higher. Since these tests run in parallel,
+            // the limit needs to be increased to avoid timeouts when running the tests.
+            ServicePointManager.DefaultConnectionLimit = int.MaxValue;
```

Comment:
Thanks.  I remember the conversation.

So, it's still not a leak, per se, but just part of the way that networking works.  It's expected that if you ask for a request stream is network bound....you get an open connection until you close it / dispose it / finalize it.

So, the choice here is a trade-off.  Add test complexity by asking for a response which does more network I/o (and might mask what we're trying to test with the request stream)....or alter the connection pool to be able to handle the parallel tests.